src/BlazorUI/Bit.BlazorUI/Components/Inputs/Slider/BitSlider.scss

```
@@ -1,451 +1,563 @@
+.bit-sld-box:has(> .bit-sld-inp-lwr:focus-visible) > .bit-sld-trk > .bit-sld-thb-upr,
+.bit-sld-box:has(> .bit-sld-inp-upr:focus-visible) > .bit-sld-trk > .bit-sld-thb-lwr {
+    box-shadow: none;
+}
 
-    input:hover::-webkit-slider-runnable-track,
-    input:active::-webkit-slider-runnable-track {
-        background: linear-gradient($BgTrackHover, $BgTrackHover) 0/var(--sx) 100% no-repeat, $BgHover;
+// The native input, laid over the track and made invisible. Its own track and thumb are stripped of every
+// appearance they have, but the thumb keeps the exact size of the drawn one so the grab area matches what
+// the eye is aiming at.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)